connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -83,29 +84,20 @@ message Expression {
     // directly declare the type variation).
     uint32 type_variation_reference = 51;
 
-    message VarChar {
-      string value = 1;
-      uint32 length = 2;
-    }
-
     message Decimal {
-      // little-endian twos-complement integer representation of complete value
-      // (ignoring precision) Always 16 bytes in length
-      bytes value = 1;
+      // the string representation.
+      string value = 1;
       // The maximum number of digits allowed in the value.
       // the maximum precision is 38.
-      int32 precision = 2;
+      optional int32 precision = 2;
       // declared scale of decimal literal
-      int32 scale = 3;
+      optional int32 scale = 3;
     }
 
-    message Map {
-      message KeyValue {
-        Literal key = 1;
-        Literal value = 2;
-      }
-
-      repeated KeyValue key_values = 1;
+    message CalendarInterval {
+      int32 months = 1;
+      int32 days = 2;
+      int64 microseconds = 3;
     }
 
     message IntervalYearToMonth {
```

Comment:
if you're removing the reference above, please remove it here as well.

Reply:
will update